plugins/reports/api/reports.js

```diff
@@ -13,7 +13,8 @@ var reportsInstance = {},
     localize = require('../../../api/utils/localization.js'),
     common = require('../../../api/utils/common.js'),
     log = require('../../../api/utils/log')('reports:reports'),
-    versionInfo = require('../../../frontend/express/version.info');
+    versionInfo = require('../../../frontend/express/version.info'),
+    offlineMode = plugins.getConfig("api").offline_mode;
```

Comment:
Here also, let's not cache value on top of file, but use `plugins.getConfig("api").offline_mode` directly where it is needed